SWLOR.Game.Server/Feature/PerkDefinition/ForcePerkDefinition.cs

```diff
@@ -450,23 +448,21 @@ private void ForceLightning()
 
         private void ForceBody()
         {
-            _builder.Create(PerkCategoryType.ForceDark, PerkType.ForceBody)
+            _builder.Create(PerkCategoryType.ForceUniversal, PerkType.ForceBody)
                 .Name("Force Body")
 
                 .AddPerkLevel()
-                .Description("Converts 25% of the user's HP into FP.")
+                .Description("Restores Willpower Mod / 2 Force Points per round.")
                 .Price(3)
                 .RequirementSkill(SkillType.Force, 20)
                 .RequirementCharacterType(CharacterType.ForceSensitive)
-                .RequirementCannotHavePerk(PerkType.ForceMind)
                 .GrantsFeat(FeatType.ForceBody1)
 
                 .AddPerkLevel()
-                .Description("Converts 50% of the user's HP into FP.")
+                .Description("Restores Willpower Mod Force Points per round.")
```

Comment:
If we keep the specific math in the description, I would reword this to something like "Restores FP equivalent to your Willpower Mod every tick."

Try to avoid the terminology for "round" because it's not dependent on the round at all. It just happens to tick at roughly the same amount of time as six seconds. And there's a possibility that it could change in the future.